.gitignore

```diff
@@ -4,3 +4,6 @@
 /build
 /django_date_extensions.egg-info/
 /.tox
+/venv
+# pyCharm
+.idea
```

Comment:
These files shouldn't be in a project .gitignore, as they're not project-specific but developer-specific; please add a global gitignore as explained on https://help.github.com/articles/ignoring-files/ and put those sort of filenames in there.